### PR TITLE
edit project

### PR DESCRIPTION
**Pull request by hys25**

### CODE REVIEW

File: src/features/project/projectSlice.js [hunk 1]
```diff
@@ -50,6 +50,18 @@ export const getProject = createAsyncThunk(
     }
   }
 )
+// PUT upfate project
```
Comment (ArturHorbenko) [inline src/features/project/projectSlice.js]: Typo here `upfate` -> `update

File: src/pages/project/ProjectCreate.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      })
+      const result = await dispatch(postProject(formData))
+      if (result.payload.status === 200) {
+        navigate(`/project/${result.payload.data._id}`)
+      }
+    },
+    [dispatch, navigate]
+  )
+  // Create <Form onSubmit={onSubmit}/>
+  // Update <Form defaultValues={defaultValues} onSubmit={onSubmit} previousMainImage={previousMainImage} previousBackgroundImage={previousBackgroundImage}/>
```
Comment (ArturHorbenko) [inline src/pages/project/ProjectCreate.js]: I guess we don't need this comment here.

File: src/pages/project/ProjectEdit.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+import { Title } from "../../elements/Title"
+import { Button } from "../../elements/Button"
+import ProjectForm from "../../layout/ProjectForm"
+import { validate } from "../../utils/validateFormInput"
+
+function ProjectEdit() {
+  const dispatch = useDispatch()
+  const navigate = useNavigate()
+  const { project } = useSelector((state) => state.project)
+  const defaultValues = {
```
Comment (ArturHorbenko) [inline src/pages/project/ProjectEdit.js]: Let's memoize `defaultValues`, so we won't recreate object on every component render.

**Review by ArturHorbenko — CHANGES_REQUESTED** [review]
Hello @hys25 , you did a great job here and decent progress. 
I made couple comments, once it'll be resolved I merge this branch.

**Comment by hys25** [comment]
Hi @ArthurGorbenko, thanks for review, pushed fix

**Comment by ArturHorbenko** [comment]
LGTM

**Pull request merged**